templates/index/recentEpochs.html

```diff
@@ -75,7 +34,7 @@ <h3 class="style-card-header d-flex justify-content-between align-items-center m
             <div class="style-paragraph-6 style-grid-header pr-2" role="columnheader">Time</div>
             <div class="style-paragraph-6 style-grid-header pr-2" role="columnheader"><span class="d-none d-sm-flex">Final</span></div>
             <div class="style-paragraph-6 style-grid-header pr-2" role="columnheader">Voted</div>
-            <div class="style-paragraph-6 style-grid-header pr-2" role="columnheader">Fee Recipient Reward</div>
```

Comment:
Why has this been changed? According to the FIGMA, "Fee Recipient Reward" was correct.

Reply:
We wan't to unify all terms for execution layer rewards (including MEV) across the site, so "EL Reward" is used for now. This is prone to change